src/index.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
```

Comment:
Styles better to import in the end of all imports. It doesn't affect anything, but looks rather clean